configure

```diff
@@ -2184,6 +2189,11 @@ else
   else
   inf "        afl-fuzz always enabled .. no"
   fi
+  if test "$with_findlib" = "true"; then
```

Comment:
You can directly

    if "${with_findlib}"; then

But really that's very minor.